src/Mod/Draft/draftgeoutils/wires.py

```diff
@@ -31,6 +31,7 @@
 
 import DraftVecUtils
 import WorkingPlane
+import FreeCAD
```

Comment:
For consistency we are importing FreeCAD as App, and then using App everywhere. These are standard abbreviations used in core files of FreeCAD.
```py
import FreeCAD as App
import FreeCADGui as Gui
```